api/data_explorer/controllers/search_controller.py

```diff
@@ -61,7 +61,7 @@ def search_get(query=None):
     # - With 1000,`api/search?query=pre` takes 7s
     num_search_results = 100
     # The number of results that Elasticsearch returns from Search queries to the fields index.
-    num_field_search_results = 10000
+    num_field_search_results = 1000
```

Comment:
Need e2e test. Change "Search box - click on drop-down row" to:
- Click search box, type "pat"
- Select first result
- assertFacet on Relationship facet
- assert "paternal grandmother" is checked

Can be separate PR.